File: Ejercicio5.js
const prompt = require('prompt-sync')(); 


let Arreglo = [1500, 2500, 3500, 4500, 5500]
Continuar = "S";

let Valor_A_Encontrar

Encontrado = null

while(Continuar.toUpperCase() === "S"){
    let Valor_A_Encontrar = parseInt(prompt("Ingrese Valor a buscar "));
    Encontrado = false

    for(let i = 0; i < Arreglo.length; i++){
        if(Arreglo[i] == Valor_A_Encontrar){
            console.log("El Valor Se ingresado se encuentra en la posicion: "+ i);
            Encontrado = true;
        }
        
    }
    if(!Encontrado){
        console.log("El valor ingresado no se encuentra en el arreglo.");
    }

    Continuar = prompt("Desea probar con otro valor? (S/N ");
}

File: Ejercicio5.cpp
#include <iostream>
using namespace std;
 
int main(){
    int Arreglo[] = {100, 200, 300, 400, 500};
    char Continuar;
    int Valor_a_Buscar;

    Continuar = 'S';
    bool encontrado;

    do{
        cout<< "Ingrese Valor a buscar" << endl;
        cin >> Valor_a_Buscar;

        encontrado = false;

        for(int i = 0; i < 5; i++){
            if(Arreglo[i] == Valor_a_Buscar){
                cout << "El Valor Se ingresado se encuentra en la posicion: " << i << endl;
               encontrado = true;
            }
        }
        
        if (!encontrado) {
            cout << "El valor ingresado no se encuentra en el arreglo." << endl;
        }   

        cout << "Desea probar con otro valor? ";
        cin >> Continuar;


    } while (Continuar == 'S');

}